### DISCUSSION TITLE
TÜRKİYE İBAX GURUBU

### DISCUSSION THREAD

**Discussion by atifileri** [IBAX-io/go-ibax#1163] [Q&A]
Soru cevap

**Comment by Celoba223** [comment]
usta IBAX Sıralama Sistemi neye göre aylık puanlar falan diyor puan kasıp yan tarafta gösteriği kadar mı alcaz ibax coini ?

**Reply by atifileri** [reply]
Evet alacağın puanlara göre cüzdana geçecek ibaxların. retweet ve her gün sandığı açmayı unutma

**Reply by Celoba223** [reply]
şuan herşeyi yaptım 26 k kadar puan var 0,5 ibax coini mi veriyor yani

**Reply by atifileri** [reply]
En yüksek puana sahip ilk 1000 kişi 500 IBXC alacak.
- 1.000'den sonrakiler 300.000 IBXC'yi eşit olarak bölüşecek.
Yani katılımcı sayısına göre 300 bin paylaşılacak.

**Reply by raintures** [reply]
paylaşımımı beğenir misiniz

**Reply by Alper10711** [reply]
son 3 günde ne yapılabilirse yapacaz bakalım ne olacak ...

**Reply by ilknurx** [reply]
benim 2 tane referansım yansımadı neden acaba

**Reply by atifileri** [reply]
Referanslar geç yansıyabilir

**Reply by Sehoo72** [reply]
Evet arkadaşlar referanslar sonradan gelebilir

**Reply by Omer5225** [reply]
türklere çıksın biraz da biz kazanalım hadi bakalım

**Comment by atifileri** [comment]
menudeki developers linkine tıklayarak puan sıralamanızı öğrenebilirsiniz.

**Comment by atifileri** [comment]
• Toplam Ödül: 5.000.000 IBXC.
• Kazanan Sayısı: 1.000 +

IBAX, gelecekteki Metaverse'in temel temeli olacak benzersiz bir Metaverse NFT madencilik sistemine sahip bir Hizmet Olarak Blockchain altyapı AĞIDIR.

Şimdi ve 1 Şubat 2022 arasında Testnet'imizin airdropunu duyurmaktan gurur duyuyoruz.

IBXC'de toplam 5.000.000 , katılımcılara airdrop olarak gönderilecek!

**Reply by atifileri** [reply]
There is a refresh problem when adding members to the group in the wallet.

**Reply by Omer5225** [reply]
yarın dağıtılacak mı peki kazandığımıza nerden bakacaz

**Comment by atifileri** [comment]
yoğunluktan dolayı şuan Retweet ödülü verilemiyor ilerleyen saatlerde tekrar deneyin puanınızı alacaksınız.

**Reply by Omer5225** [reply]
saat 16 gibi düzelir diyorlar
